### ISSUE TITLE
Question: Unclear Allow unprotected path option?

### ISSUE THREAD

**Issue by ldeveloperl1985** [GluuFederation/gluu-gateway#250]
![cc](https://user-images.githubusercontent.com/2329776/47638918-ff96b400-db85-11e8-882f-877adbcd60a1.png)

> Mike: It’s also really unclear what this means…
If it’s not specified, it’s denied by default ?

**Comment by ldeveloperl1985** [comment]
@nynymike Yes correct it's denied by default. By default value is NO(false). 
1. If Yes, then allow unprotected path
2. If No, then denied unprotected path

**Comment by altexy** [comment]
`Allow Unprotected path` options is related to `Allow expression`.
It make sense only when `Allow expression` is `Yes`, because `unprotected path` means any path not listed in scope expression.

IMO, We need to place `Allow Unprotected path` below `Allow expression` and auto disable it when `Allow expression` is `No`

**Comment by ldeveloperl1985** [comment]
Agreed, I'll changed it

**Comment by ldeveloperl1985** [comment]
@altexy update the field as suggested by mike. I'll update it from UI side.
1. In Both plugins, change `allow_unprotected_path` field name to `deny_by_default`. Default value should be "true"
2. In OAUTH-PEP, change `allow_oauth_scope_expression` field name to `ignore_scope`. Default value should be "false".

**Comment by altexy** [comment]
@ldeveloperl1985 
`ignore_scope` - I don't understand this.
When it is `true` should we check scope expression or not?

**Comment by ldeveloperl1985** [comment]
1. `ignore_scope = false`, plugin **will check** scope expression. By default it is `false`.
2. `ignore_scope = true`, plugin **will not check** scope expression. it will ignore the scopes.
